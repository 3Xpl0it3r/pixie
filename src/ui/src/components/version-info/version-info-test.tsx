import * as React from 'react';
import { render } from 'enzyme';
import VersionInfo from './version-info';

describe('<VersionInfo/>', () => {
  it('renders correctly', () => {
    const wrapper = render(<VersionInfo cloudVersion='testing 123' />);
    expect(wrapper).toMatchSnapshot();
  });
});
