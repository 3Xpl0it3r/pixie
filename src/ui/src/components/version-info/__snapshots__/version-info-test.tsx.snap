// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VersionInfo/> renders correctly 1`] = `
<div>
  <div
    class="VersionInfoImpl-root-1"
  >
    testing 123
  </div>
</div>
`;
